packages/web-app/src/components/Checkbox.tsx

```diff
@@ -116,6 +128,7 @@ class _Checkbox extends Component<Props> {
         >
           {!hideCheckbox && (
             <div
+              style={checked ? { backgroundColor: checkedColor } : undefined}
```

Comment:
Why do you need to use style and not className?

Reply:
`checkedColor` is a dynamic value, what is your suggestion, could you share the code, please?

Reply:
Add checkBoxClassName property and put it in className

```suggestion
              className={classnames(classes.checkBox, checkBoxClassName, { [classes.textElementEnabled]: hasTextElement })}
```

Reply:
looks, more complex to me, and requires more code to implement and use

Reply:
Agree with, let's stay with style